docs/config.md

```diff
@@ -69,9 +69,7 @@ in their representations.
 ```python
 >>> from myproject import settings
 >>> settings.DATABASE_URL
-DatabaseURL('postgresql://admin:**********@192.168.0.8/my-application')
->>> str(settings.DATABASE_URL)
-'postgresql://admin:Fkjh348htGee4t3@192.168.0.8/my-application'
+'postgresql://user:password@localhost:5432/database'
```

Comment:
Is this right? Isn't password be *******?

Reply:
Yes that's correct, databases did that but SQLALchemy engine accepts pure string

Reply:
But then this section becomes useless, doesn't it? :sweat_smile:

Reply:
Then... :eyes:

Reply:
Ah that's what you meant, I'll remove that.